config/locales/en.yml

```diff
@@ -648,6 +648,8 @@ en:
           pets: Pets
           applications: Applications
           rescues: Rescues
+          header: "Welcome to Beja Pet Rescue!"
+          body_text: "At Baja Pet Rescue, we’re dedicated to rescuing, rehabilitating, and rehoming animals in need. Your support helps us provide care, love, and hope to these pets until they find their forever homes."
```

Comment:
```suggestion
          header: "Welcome to %{organization}!"
          body_text: "At %{organization}, we’re dedicated to rescuing, rehabilitating, and rehoming animals in need. Your support helps us provide care, love, and hope to these pets until they find their forever homes."
```

Reply:
This ^ these should be dynamically generated